interface/forms/newpatient/common.php

```diff
@@ -223,6 +223,16 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
                   continue;
               }
 
+              // Fetch acl for category of given encounter. Only if has write auth for a category, then can create an encounter of that category.
+              $postCalendarCategoryACO = fetchPostCalendarCategoryACO($catId);
+              $postCalendarCategoryACO = explode('|',$postCalendarCategoryACO);
```

Comment:
done
(though I personally like to break things up because to me it's clearer that way)